### PR TITLE
Quarter4 pipeline

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configurations/pipeline_config.json [hunk 1]
```diff
@@ -133,6 +133,7 @@
     {"RapidProKey": "Young_People (Value) - mag_s01_w10_follow_up", "PipelineKey": "young_people_issues_raw"},
     {"RapidProKey": "Young_People (Run ID) - mag_s01_w10_follow_up", "PipelineKey": "young_people_issues_run_id"}
   ],
+  "PipelineName": "main_pipeline",
```
Comment (as2388) [inline configurations/pipeline_config.json]: If this is for the 'main_pipeline', or 'all_quarters', why don't we name this file as such

File: generate_outputs.py [hunk 2]
```diff
@@ -153,7 +153,16 @@
...
     data = CombineRawDatasets.combine_raw_datasets(user, messages_datasets + recovery_datasets, coalesced_follow_up_datasets,
                                                    coalesced_demog_datasets)
-    
+
+    # Infer which RQA coding plans to use from the pipeline name.
+    if pipeline_configuration.pipeline_name == "quarter_four":
+        log.info("Running quarter four pipeline")
+        PipelineConfiguration.RQA_CODING_PLANS = PipelineConfiguration.QUARTER_FOUR_RQA_CODING_PLANS
+    else:
+        assert pipeline_configuration.pipeline_name == "main_pipeline", "PipelineName must be either 'a quartely pipeline name' or 'main pipeline'"
```
Comment (as2388) [inline generate_outputs.py]: Consider also putting a check in PipelineConfiguration.validate

File: src/lib/pipeline_configuration.py [hunk 3]
```diff
@@ -98,7 +98,9 @@ class PipelineConfiguration(object):
     # TODO revise this as the project nears the end
     PROJECT_END_DATE = isoparse("2020-02-15T09:00:00+03:00")
 
-    RQA_CODING_PLANS = [
+    RQA_CODING_PLANS = None
+
+    MAIN_RQA_CODING_PLANS = [
```
Comment (as2388) [inline src/lib/pipeline_configuration.py]: Consider ALL_RQA_CODING_PLANS or ALL_QUARTERS...

File: configurations/pipeline_config.json (hunk 1 above)
Comment (IsaackMwenda) [reply configurations/pipeline_config.json]: Updated thanks!

File: src/lib/pipeline_configuration.py (hunk 3 above)
Comment (IsaackMwenda) [reply src/lib/pipeline_configuration.py]: Updated thanks!

**Pull request merged**